.github/workflows/flutter.yml

```diff
@@ -0,0 +1,71 @@
+name: "Flutter build"
+on:
+  release:
+    types: [published]
+    branches: ['app']
+  workflow_dispatch:
+jobs:
+  linux_build:
+    name: "Linux build"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout repo"
+        uses: actions/checkout@v3
+      - name: "Setup flutter"
+        uses: subosito/flutter-action@v2
+        with:
+          flutter-version: '3.3.3'
+          architecture: x64
+      - name: "Install dependencies"
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y ninja-build libgtk-3-dev
+      - name: "Build"
+        run: |
+          flutter config --enable-linux-desktop
+          flutter build linux --release --no-sound-null-safety
+      - name: Archive production artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: lb-planer-linux
+          path: build/linux/x64/release/bundle/*
+  windows_build:
+    name: "Windows build"
+    runs-on: windows-latest
+    steps:
+      - name: "Checkout repo"
+        uses: actions/checkout@v3
+      - name: "Setup flutter"
+        uses: subosito/flutter-action@v2
+        with:
+          flutter-version: '3.3.3'
+          architecture: x64
+      - name: "Build"
+        run: |
+          flutter config --enable-windows-desktop
+          flutter build windows --release --no-sound-null-safety
+      - name: Archive production artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: lb-planer-windows
+          path: build\windows\runner\Release\*
+#   macos_build:
```

Comment:
mark the PR as a draft until you can uncomment the macos build please

Reply:
I'm sorry but no, since it can take too long to test the implementation on this platform (since it hasn't ever been tested) and the workflow works fine even without it. MacOS section can be changed anytime

Reply:
imo macos is not really a priority right now and can be added later